ui/apps/platform/src/services/RolesService.ts

```diff
@@ -116,3 +117,53 @@ export function updatePermissionSet(entity: PermissionSet): Promise<Empty> {
 export function deletePermissionSet(id: string): Promise<Empty> {
     return axios.delete(`${permissionSetsUrl}/${id}`);
 }
+
+const clustersForPermissionsUrl = '/v1/sac/clusters';
```

Comment:
As I currently understand this feature, we are hoping to reuse this endpoint for many other data types that need to be retrieved, even though the user may not have full permissions for them.

Should we make the endpoint something besides "clusters" ? 

Otherwise, there will be a proliferation of sub-endpoints under the `sac/` root.

Reply:
In the current state of things, the `/v1/sac/` endpoint should only have two sub-endpoints: `/v1/sac/clusters` and `/v1/sac/clusters/{cluster_id}/namespaces`.
I think the current state of the endpoint and its sub-endpoint reflects the state of access scopes in the product and should not evolve more at this point. Future evolutions of the access scope model may have to be reflected in the sub-endpoint hierarchy.